static/js/components/dashboard/CoursePrice.js

```diff
@@ -11,25 +16,79 @@ import { formatPrice } from '../../util/util';
 
 export default class CoursePrice extends React.Component {
   props: {
-    course: Course,
+    course: Course
   };
 
+  renderTooltip(text: string): React$Element<*>|void {
+    let tooltip;
+
+    if (text) {
+      tooltip = (
+        <div>
+          <span className="tooltip-link"
+            data-tip
+            data-for='course-detail'>
+            <IconButton name="help" className="help"/>
+          </span>
+          <ReactTooltip id="course-detail" effect="solid"
+            event="click" globalEventOff="click" className="tooltip">
+            {text}
+          </ReactTooltip>
+        </div>
+      );
+    }
+
+    return tooltip;
+  }
+
+  courseTooltipText(courseStatus: string): string {
+    let text = "";
+
+    if (courseStatus === STATUS_ENROLLED_NOT_VERIFIED) {
+      text = "You need to enroll in the Verified Course to get MicroMasters credit.";
+    }
```

Comment:
this would be more concise

```javascript
if (courseStatus === STATUS_ENROLLED_NOT_VERIFIED) {
  return "You need to enroll in the Verified Course to get MicroMasters credit.";
}
return '';
```